src/crimson/os/seastore/seastore_types.h

```diff
@@ -99,30 +194,37 @@ constexpr record_delta_idx_t NULL_DELTA_IDX =
  * record_relative paddrs.
  */
 struct paddr_t {
-  segment_id_t segment = NULL_SEG_ID;
+  device_segment_t segment = NULL_DEVICE_SEGMENT;
```

Comment:
This is an example of what I meant above.  I'd really prefer to leave segment_id_t and NULL_SEG_ID here and give the other usage a new type/constants.

Reply:
ok.